packages/lesswrong/components/shortform/ShortformSubmitForm.jsx

```diff
@@ -4,14 +4,18 @@ import { Components, registerComponent } from 'meteor/vulcan:core';
 
 const ShortformSubmitForm = ({currentUser, successCallback}) => {
   const { CommentsNewForm } = Components;
+  const shortformFeedId = currentUser?.shortformFeedId
+
   return (
     <CommentsNewForm
+      post={{_id:shortformFeedId}} 
```

Comment:
I took that out deliberately. The API is that matching up to the shortform feed container is done server side.